analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/ConditionEvaluatesToConstant.CSharp8.cs

```diff
@@ -392,15 +392,15 @@ void Method()
                 continue;
             }
 
-            if (sb is null) // Noncompliant FP
+            if (sb is null)     // Noncompliant FP
             {
                 sb = new StringBuilder();
             }
 
             sb.Append(i);
         }
 
-        if (sb is null) // Noncompliant FP
+        if (sb is null) // FN
```

Comment:
FP cannot turn into FN

Reply:
The FP was an error. This used to be a TP (found by pure chance) which we unfortunately lost.